src/ui/gui.css

```diff
@@ -286,8 +286,17 @@ select.center {
 
 .die_overlay {
     position:relative;
-    font-size: 24px;
-    top: 13px;
+    font-size: 21px;
+    top: 14px;
+}
+
+/* Added slightly different vertical spacing for Firefox only */
+@-moz-document url-prefix() {
+    .die_overlay{
+        position:relative;
+        font-size: 21px;
+        top: 17px;
+    }
```

Comment:
I'm pretty sure that I could have left off the repeated stuff, but I didn't test with it. Anyway, hopefully we won't need this override, based on your suggestion below. Testing now.